modules/ftrackIdSystem.md

```diff
@@ -0,0 +1,87 @@
+# Flashtalking's FTrack Identity Framework User ID Module
+
+*The FTrack Identity Framework User ID Module allows publishers to take advantage of Flashtalking's FTrack ID during the bidding process.*
+
+**THE ONLY COMPLETE SOLUTION FOR COOKIELESS MEASUREMENT & PERSONALIZATION**
+
+Flashtalking is the world’s first ad serving platform to function without cookies to orchestrate client identity across buy-side ID spaces for measurement and personalization. With over 120 active global advertisers, our cookieless identity framework is market-ready and includes privacy controls to ensure consumer notification and choice on every impression.
+
+### [FTrack](https://www.flashtalking.com/identity-framework#FTrack)
+
+Flashtalking’s cookieless tracking technology uses probabilistic device recognition to derive a privacy-friendly persistent ID for each device.
+
+**PROVEN**  
+With over 120 brands using FTrack globally, Flashtalking has accumulated the largest cookieless footprint in the industry.
+
+**ANTI-FINGERPRINTING**  
```

Comment:
Most of the code appears to be an attempt to form a fingerprint. The RTD module you provide send user segments. Is anything in the paragraph below accurate?

Reply:
@patmmccann 

Flashtalking's FTrack ID should not be considered fingerprinting software since Fingerprinting, as defined by Google, includes opaque or hidden techniques, tracking in a covert manner, and identity creation without user knowledge or their ability to opt out. 

- Flashtalking deploys a privacy icon (purple shield) on every ad served with FTrack enabled. This icon when moused over announces itself as a "Privacy Notification" and links the user directly to information about what Flashtalking and FTrack are, and aren't.  
- This Consumer Privacy page also links to Flashtalking's privacy policy and opt-out, as well as the AdChoices opt-out and the EDAA's global opt-out site.  
- Additionally, we leverage the FTrack ID to maintain a user's opt-out without reliance on cookies, unlike other vendors who offer "cookie-based" opt-out only for cookieless trackers.  
- Importantly, FTrack does not access PII, including email addresses, nor any sensitive information. 
- The negative concepts of fingerprinting are contingent upon the avoidance of user transparency and opt out. Flashtalking contends that leveraging low-risk identity signals while actively informing users of what their data is being used for, and providing easy, persistent opt-out, is a far more user and privacy-centric system.  

We feel it is extremely important to inform potential partners of the difference between fingerprinting as an opaque, hidden, covert tracking system without user knowledge and transparency and FTrack as a consent-driven, privacy-friendly probabilistic ID. We welcome the opportunity to walk through exactly what FTrack can, does, and doesn't do in terms of user data and our extensive system-wide privacy tools designed to maximize transparency around our tools to users on the open web.

Reply:
We discussed this in committee and we're not comfortable with the collect signals functionality of your code. You have a couple options that could lead to a merge. Many of your peer identity providers may or may not collect these signals in code that is not linked from or contained within this library eg #7985 provides a reference implementation of both assuming that the publisher has integrated already and directly with the identity provider, and a way that facilitates an integration without referencing the script directly, but making it a configuration parameter.

Reply:
Thanks @patmmccann . I shared your feedback with our "committee" and we will make adjustments based on your feedback.

Reply:
@patmmccann The config and code have been changed. Hopefully the change successfully addresses the points the committee made about the previous commit. Thanks